### DISCUSSION TITLE
How to fix wrong image resizing

### DISCUSSION THREAD

**Discussion by Alex2021cyberhuman** [SixLabors/ImageSharp#1741] [Q&A] [answered]
I tried to resize windows screenshots.
Resize Options
```
image.Mutate(configureOptions, mutation => mutation.Resize(options: new()
{
    Size = new(200, 200),
    Mode = ResizeMode.Crop,
    Position = AnchorPositionMode.Center
}));
```

Saving to file. Receive file from IFormFile asp net core

```
await using var formStream = formFile.OpenReadStream();
var (image, format) = await Image.LoadWithFormatAsync(configureOptions, formStream, cancellationToken);
if (format is null || image is null)
{
    ModelState.AddModelError(string.Empty, ErrorCodes.InvalidFormat);
    return BadRequest(ModelState);
}
```

Original: 
![screen](https://user-images.githubusercontent.com/59359082/129561681-5ed958b2-d7fa-4122-863a-c0e319c70f7c.png)
Result:
![result](https://user-images.githubusercontent.com/59359082/129561716-b767a010-65c4-4ed7-ad34-22c3f9283649.png)

***Thanks in advance***

**Comment by Lure5134** [comment]
I have the same issue with the 1.0.3 version (newest). I have checked the project out and on the newest master it works all great.
@JimBobSquarePants can you release a new version so we can use it?

**Comment by JimBobSquarePants** [comment, marked as answer]
You can use the MyGet feed. We’re working to deliver V2 in the next two months
